units/commit_queue_test.go

```diff
@@ -127,42 +125,6 @@ func (s *commitQueueSuite) TestNewCommitQueueJob() {
 	s.Equal("commit-queue:mci_job-1", job.ID())
 }
 
-func (s *commitQueueSuite) TestWritePatchInfo() {
```

Comment:
these 2 tests were moved to https://github.com/evergreen-ci/evergreen/pull/4408/files#diff-9e2146d77665f5b3af3de33ac185dcdce9f6a17d2edbda1c968f0a2a2b237326R357